ONSdigital/census-rm-acceptance-tests-2021
# Smoke test wip files produced

# Motivation and Context
There wasn't a test to check that a correct notification file was actually created from creating a collex and uploading a file.
This has been added now along with some alterations to the sample-stub-svc to make the system produce files

# What has changed
New @smoketest with
New  step to check that files containing the expected data are produced and on the sftp server

# How to test?
To produce files locally you need the branch: https://github.com/ONSdigital/census-rm-docker-dev/pull/4 of docker-dev (this is also in PR, so may make it to master itself)

Then run 'make up' in census-rm-docker-dev to recreate the sample-stub-svc

get this branch of census-rm-acceptance-tests
run pipenv install --dev . 
run 'make test'
The tests should work and on the local sftp server the files should exists
 docker exec -it sftp /bin/bash
 cd home/centos/Documents/sftp/

 *once you've updates the sample svc with the correct ports all your previous cases in the DB/Queues will flow through, so there could be multiple files.  You might want to delete these is you want to explicitly check the tests

# Notes:
The settings for getting this to work with k8s sftp aren't all complete yet as, waiting for advice

# Links
https://trello.com/c/fjQusTOt/530-smoke-test-for-sample-load-print-generation
